.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,12 @@
+Thank you for taking the time to work on a PR for Awesome-selfhosted!
+
+To ensure your PR is dealt with swiftly please check the following:
+
+- [ ] Your submissions are formatted according to the guidelines. 
+        
+    ``[Name](http://homepage/) - Short description. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code)) `License` `Language```
```

Comment:
[Name](http://homepage/) - Short description. ([Demo](http://url.to/demo), [Example](http://url.to/example), [Source Code](http://url.of/source/code)) `License` `Language`